djl-serving/java-client/src/main/java/ai/djl/examples/serving/javaclient/HttpUtils.java

```diff
@@ -0,0 +1,60 @@
+/*
```

Comment:
Can we upgrade to Java 11 and use Java built-in HttpClient?

Reply:
Done.